grizli/pipeline/default_params.py

```diff
@@ -14,6 +14,8 @@
 IR_M_FILTERS = ['F098M', 'F127M', 'F139M', 'F153M']
 IR_W_FILTERS = ['F105W', 'F110W', 'F125W', 'F140W', 'F160W']
 
+JW_FILTERS = ['F115W', 'F150W', 'F200W']
```

Comment:
There are more filters, and can also add, e.g., `NIRCAM_LW_FILTERS = ...` and `NIRCAM_SW_FILTERS = ...`.   And eventually `MIRI_FILTERS`, but maybe that can wait until other MIRI logic is implemented.

Reply:
I've added imaging filters for NIRISS and NIRCAM here: 46d54879fc65c9a45098b391f5e02fa957523f18

But how do I list the NIRCam grisms?

Reply:
Added MIRI filters here: e59c0b7c85120da783ef7348939772292e0a233f